tern_api/api/v1/commom_models.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
+#
```

Comment:
file is named `commom_models`.. is this an api specific term or should it be `common_models`?